data/80-udisks2.rules

```diff
@@ -122,12 +122,14 @@ ENV{ID_PART_ENTRY_SCHEME}=="mac", ENV{ID_PART_ENTRY_TYPE}=="Apple_Bootstrap", EN
 
 # Apple Boot partitions
 ENV{ID_PART_ENTRY_SCHEME}=="gpt", ENV{ID_PART_ENTRY_TYPE}=="426f6f74-0000-11aa-aa11-00306543ecac", ENV{UDISKS_IGNORE}="1"
+ENV{ID_FS_LABEL}=="ANACONDA", ENV{ID_PART_ENTRY_TYPE}=="48465300-0000-11aa-aa11-00306543ecac|0xaf", ENV{UDISKS_IGNORE}="1"
```

Comment:
I'm not sure if it's a good idea to put something distribution specific in upstream rules, this could quickly go out of hand if every distribution wanted to add a specific rule here. Maybe LiveCD tools can ship their own UDev rule?

Reply:
I have no problem including this downstream only (even within the `udisks2` package) under extra udev rules file. As for inclusion upstream I'm only about 50% sure/unsure. (The `80-udisks2.rules` file is messy anyway and most of the rules should be pushed further to lower levels).